neo4j/metadata.csv

```diff
@@ -1,119 +1,349 @@
-neo4j.transaction.terminated_read,gauge,,,,The total number of terminated read transactions.,0,neo4j,transaction terminated read,
-neo4j.transaction.terminated,gauge,,,,The total number of terminated transactions.,0,neo4j,transaction terminated,
-neo4j.transaction.terminated_write,gauge,,,,The total number of terminated write transactions.,0,neo4j,transaction terminated write,
-neo4j.vm.gc.count.g1_old_generation,gauge,,,,Total number of garbage collections for old generation.,0,neo4j,vm gc count g1 old generation,
-neo4j.vm.gc.count.g1_young_generation,gauge,,,,Total number of garbage collections for young generation.,0,neo4j,vm gc count g1 young generation,
-neo4j.vm.gc.time.g1_old_generation,gauge,,,,Accumulated garbage collection time in milliseconds for old generation.,0,neo4j,vm gc time g1 old generation,
-neo4j.vm.gc.time.g1_young_generation,gauge,,,,Accumulated garbage collection time in milliseconds for young generation.,0,neo4j,vm gc time g1 young generation,
-neo4j.vm.memory.buffer.direct_capacity,gauge,,,,Estimated total capacity of direct buffers in the pool.,0,neo4j,vm memory buffer direct capacity,
-neo4j.vm.memory.buffer.direct_count,gauge,,,,Estimated number of direct buffers in the pool.,0,neo4j,vm memory buffer direct count,
-neo4j.vm.memory.buffer.direct_used,gauge,,,,Estimated amount of direct memory used by the pool.,0,neo4j,vm memory buffer direct used,
-neo4j.vm.memory.buffer.mapped_capacity,gauge,,,,Estimated total capacity of mapped buffers in the pool.,0,neo4j,vm memory buffer mapped capacity,
-neo4j.vm.memory.buffer.mapped_count,gauge,,,,Estimated number of mapped buffers in the pool.,0,neo4j,vm memory buffer mapped count,
-neo4j.vm.memory.buffer.mapped_used,gauge,,,,Estimated amount of mapped memory used by the pool.,0,neo4j,vm memory buffer mapped used,
-neo4j.vm.memory.pool.compressed_class_space,gauge,,,,Estimated number of buffers in the compressed class space pool.,0,neo4j,vm memory pool compressed class space,
-neo4j.vm.memory.pool.g1_eden_space,gauge,,,,Estimated number of buffers in the g1 eden space pool.,0,neo4j,vm memory pool g1 eden space,
-neo4j.vm.memory.pool.g1_old_gen,gauge,,,,Estimated number of buffers in the g1 old generation pool.,0,neo4j,vm memory pool g1 old gen,
-neo4j.vm.memory.pool.g1_survivor_space,gauge,,,,Estimated number of buffers in the g1 survivor space pool.,0,neo4j,vm memory pool g1 survivor space,
-neo4j.vm.memory.pool.metaspace,gauge,,,,Estimated number of buffers in the metaspace pool.,0,neo4j,vm memory pool metaspace,
-neo4j.vm.thread.count,gauge,,,,Estimated number of active threads in the current thread group.,0,neo4j,vm thread count,
-neo4j.vm.thread.total,gauge,,,,The total number of live threads including daemon and non-daemon threads.,0,neo4j,vm thread total,
+neo4j.dbms.bolt.accumulated_processing_time,count,,,,The total amount of time in milliseconds that worker threads have been processing messages.,0,neo4j,bolt accumulated processing time,
```

Comment:
I see that you are changing all of the metrics in metadata.csv with this new version. For customers running the older versions of the check, they may be confused by what metrics this check is supposed to emit (especially since you are also adding a v4 dashboard in this PR), so I think you should do one of the two options:
1. include in your README a link to the original metadata.csv for v4 (and maybe any previously changed metadata.csv's)
2. Keep the v4 metrics but add a `[v4 only]` or something similar and add `[v5 only]` to your new metrics.

Reply:
I'll take option 2 and resubmit.

Reply:
Updated with option 2.